packages/opentelemetry-metrics/src/types.ts

```diff
@@ -28,9 +28,6 @@ export interface MetricOptions {
   /** The unit of the Metric values. */
   unit: string;
 
```

Comment:
Why are there two `type` files for metrics? One here and one in opentelemetry-types?

Reply:
`opentelemetry-types` contain API level definitions, where SDK level is added here.

Reply:
It seems like `MetricOptions` are in both files? The one in the sdk just has an extra `logger` field. Is this on purpose?

Reply:
Yes, this is intentional in order to get ride of [Non-Null Assertion Operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html)`!`. 

Related comment: https://github.com/open-telemetry/opentelemetry-js/pull/415#discussion_r333072868